packages/suite/src/components/suite/AccountLabel.tsx

```diff
@@ -16,27 +17,47 @@ export interface AccountLabelProps {
     index?: number;
 }
 
+export const useAccountLabel = () => {
+    const { translationString } = useTranslation();
+
+    const defaultAccountLabelString = useCallback(
+        ({
+            accountType,
+            symbol,
+            index = 0,
+        }: {
+            accountType: Account['accountType'];
+            symbol: Account['symbol'];
+            index?: number;
+        }) => {
+            if (accountType === 'coinjoin') {
+                return translationString(getTitleForCoinjoinAccount(symbol));
+            }
+
+            return translationString('LABELING_ACCOUNT', {
+                networkName: translationString(getTitleForNetwork(symbol)), // Bitcoin, Ethereum, ...
+                index: index + 1, // this is the number which shows after hash, e.g. Ethereum #3
+            });
+        },
+        [translationString],
+    );
+
+    return {
+        defaultAccountLabelString,
+    };
+};
+
 export const AccountLabel = ({
     accountLabel,
     accountType,
     symbol,
     index = 0,
 }: AccountLabelProps) => {
+    const { defaultAccountLabelString } = useAccountLabel();
+
     if (accountLabel) {
         return <TabularNums>{accountLabel}</TabularNums>;
     }
 
-    if (accountType === 'coinjoin') {
-        return <Translation id={getTitleForCoinjoinAccount(symbol)} />;
-    }
-
-    return (
-        <Translation
-            id="LABELING_ACCOUNT"
-            values={{
-                networkName: <Translation id={getTitleForNetwork(symbol)} />, // Bitcoin, Ethereum, ...
-                index: index + 1, // this is the number which shows after hash, e.g. Ethereum #3
-            }}
-        />
-    );
+    return <>{defaultAccountLabelString({ accountType, symbol, index })}</>;
```

Comment:
Is fragment neede here?

Reply:
Yes, to make the return type to be React.Element
![image](https://github.com/trezor/trezor-suite/assets/152899911/a2928c8b-72dc-4ad9-829f-17512700a908)